libs/datamodel/connectors/sql-datamodel-connector/src/mysql_datamodel_connector.rs

```diff
@@ -427,6 +456,6 @@ impl Connector for MySqlDatamodelConnector {
 
 impl Default for MySqlDatamodelConnector {
     fn default() -> Self {
-        Self::new()
+        Self::new(false)
```

Comment:
It looks like this could bite :/ Where is that default impl used?

Reply:
removing this.